openassessment/xblock/test/test_staff_area.py

```diff
@@ -494,7 +528,7 @@ def test_schedule_training(self, xblock):
         # Schedule training
         response = self.request(xblock, 'schedule_training', json.dumps({}), response_format='json')
         self.assertTrue(response['success'], msg=response.get('msg'))
-        self.assertTrue('workflow_uuid' in response)
+        self.assertIn('workflow_uuid', response)
```

Comment:
The test I added above bumped our pylint violation count up by 1, so I fixed this easy violation to compensate and keep the autmoated tests happy.